pkg/commands/oscommands/os_test_default.go

```diff
@@ -49,21 +53,23 @@ func TestOSCommandOpenFileDarwin(t *testing.T) {
 		oSCmd.Platform.OS = "darwin"
 		oSCmd.UserConfig.OS.OpenCommand = "open {{filename}}"
 
-		s.test(oSCmd.OpenFile(s.filename))
```

Comment:
can we add a test verifying that if we've configured OpenCommandTemplate that the file number does indeed come through as expected?